src/Modules/Microsoft.Azure.Functions.PowerShellWorker/Microsoft.Azure.Functions.PowerShellWorker.psd1

```diff
@@ -61,7 +61,7 @@ CmdletsToExport = @(
     'Set-FunctionInvocationContext',
     'Start-DurableTimer',
     'Trace-PipelineObject',
-    'Wait-ActivityFunction')
+    'Wait-DurableTask')
```

Comment:
Let's create an alias mapping the old name to the new one, so that we don't break existing code. We'll need something like `Set-Alias -Name Wait-ActivityFunction -Value Wait-DurableTask` in the .psm1, and add the alias name to the AliasesToExport entry below.

Reply:
Sounds good. I will then revert the changes to the TestFunctionApp orchestrators and the sample code where I replaced Wait-ActivityFunction with Wait-DurableTask.

Reply:
See the comment at https://github.com/Azure/azure-functions-powershell-worker/pull/518#discussion_r473213770